lib/run.js

```diff
@@ -91,8 +92,12 @@ function run (opts, cb) {
     client.on('response', onResponse)
     client.on('connError', onError)
     client.on('timeout', onTimeout)
+    client.on('request', () => { totalRequests++ })
     client.on('done', onDone)
     clients.push(client)
+
+    // we will miss the initial request emits because the client emits request on construction
+    totalRequests += url.pipelining < url.rate ? url.rate : url.pipelining
```

Comment:
I could also trigger the initial request batch by putting it in a nextTick, if you'd prefer